packages/@expo/cli/src/utils/multipartMixed.ts

```diff
@@ -0,0 +1,51 @@
+import { randomBytes } from 'node:crypto';
+
+export interface FormDataField {
+  name: string;
+  value: string | File | Blob;
+  contentType?: string | null;
+  partHeaders?: Record<string, string> | null;
+}
+
+export interface EncodedFormData {
+  boundary: string;
+  body: string;
+}
+
+const CRLF = '\r\n';
+const BOUNDARY = '-'.repeat(2);
```

Comment:
```suggestion
const BOUNDARY_HYPHEN_CHARACTERS = '-'.repeat(2);
```
(using nomenclature from the rfc)
https://www.ietf.org/rfc/rfc2046.txt#:~:text=entirely%20of%20two-,hyphen%20characters,-(%22%2D%22%2C%20decimal%20value%2045

Reply:
much better! 🙌 glad the spec has an answer for what it's called